app/frontend/entrypoints/application.scss

```diff
@@ -1,16 +1,35 @@
+// Modules
+
+@import "../stylesheets/modules/colors";
+@import "../stylesheets/modules/fonts";
+
+// Libraries
+
 @import "~bootstrap/scss/bootstrap";
 @import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css");
 
+// Bootstrap overrides
+@import "../stylesheets/bootstrap/navbar";
+@import "../stylesheets/bootstrap/breadcrumb";
+
 // Mixins
-@import "../stylesheets/mixins/bem";
 
-// Modules
-@import "../stylesheets/modules/colors";
+@import "../stylesheets/mixins/bem";
 
 // Blocks
-@import "../stylesheets/blocks/icons";
+
+@import "../stylesheets/blocks/card";
+@import "../stylesheets/blocks/form-in-dropdown";
+@import "../stylesheets/blocks/header";
 @import "../stylesheets/blocks/record-view";
 
+
```

Comment:
Maybe consider moving this to its own file

Reply:
Done :)